esp/scm/ws_logaccess.ecm

```diff
@@ -0,0 +1,77 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2021 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+ESPenum LogAccessType : int
+{
+    All(0, "All"),
+    ByWUID(1, "ByWUID"),
+    ByComponent(2, "ByComponent"),
+    ByLogType(3, "ByLogType"),
+    ByTargetAudience(4, "ByTargetAudience")
+};
+
+ESPenum LogAccessLogFormat : int
+{
+    XML(0, "XML"),
+    JSON(1, "JSON"),
+    CSV(2, "CSV")
+} ;
+
+ESPStruct TimeRange
+{
+    xsdDateTime StartDate; //example 1980-03-23T10:20:15
+    xsdDateTime EndDate;   //example 1980-03-23T10:20:15
+};
+
+ESPRequest GetLogAccessMetaRequest
+{
+};
+
+ESPResponse GetLogAccessMetaResponse
+{
+    string RemoteLogManagerType;
+    string RemoteLogManagerConnectionString;
+};
+
+ESPRequest GetLogsRequest
+{
+    ESPenum LogAccessType LogCategory;
+    string SearchByValue;
+    ESPStruct TimeRange Range;
+    int LogLineLimit(100);
+    int LogLineOffset(0);
+    ESParray<string> Columns;
+    LogAccessLogFormat Format("JSON");
+};
+
+ESPResponse GetLogsResponse
+{
+    string LogLines;
+    boolean Success;
+    string Message;
```

Comment:
Should this only be an error message (I don't know what the convention for other esp services is.)  Does ESPresponse already cover this?

Reply:
Un-handled error messages are reported via soapfaults, a lot of service throw any/all error.
Several other services utilize the message field coupled w/ the success bool, I believe to communicate handled errs and/or warns/info.